### ISSUE TITLE
The HTTP interface broke

### ISSUE THREAD

**Issue by goncalotomas** [goncalotomas/FMKe#113]
... And if #62 was implemented this wouldn't have ended up in `master`. Welp.

Time to fix #62.

Reason of crash:
```erlang
{[{reason,undef},{mfa,{fmke_http_handler_patients,init,3}},{stacktrace,[{fmke_http_handler_patients,init,[{tcp,http},{http_req,#Port<0.1503>,ranch_tcp,keepalive,<0.703.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},54475},<<"127.0.0.1">>,undefined,9090,<<"/patients/1">>,undefined,<<>>,undefined,[{id,<<"1">>}],[{<<"content-length">>,<<"0">>},{<<"te">>,<<>>},{<<"host">>,<<"127.0.0.1:9090">>},{<<"connection">>,<<"keep-alive">>}],[{<<"connection">>,[<<"keep-alive">>]}],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined},[]],[]},{cowboy_handler,handler_init,4,[{file,"/private-directory/_build/default/lib/cowboy/src/cowboy_handler.erl"},{line,64}]},{cowboy_protocol,execute,4,[{file,"/private-directory/_build/default/lib/cowboy/src/cowboy_protocol.erl"},{line,442}]}]},{req,[{socket,#Port<0.1503>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.703.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},54475}},{host,<<"127.0.0.1">>},{host_info,undefined},{port,9090},{path,<<"/patients/1">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[{id,<<"1">>}]},{headers,[{<<"content-length">>,<<"0">>},{<<"te">>,<<>>},{<<"host">>,<<"127.0.0.1:9090">>},{<<"connection">>,<<"keep-alive">>}]},{p_headers,[{<<"connection">>,[<<"keep-alive">>]}]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{opts,[]}],[{cowboy_handler,handler_init,4,[{file,"/private-directory/_build/default/lib/cowboy/src/cowboy_handler.erl"},{line,89}]},{cowboy_protocol,execute,4,[{file,"/private-directory/_build/default/lib/cowboy/src/cowboy_protocol.erl"},{line,442}]}]}
```

**Comment by goncalotomas** [comment]
Cause is the "update" to hex packages where cowboy was left at `1.1.2`, and we were using the `2.0.0` interface already.

**Comment by goncalotomas** [comment]
This is going to be fixed once #103 is merged.
